### PR TITLE
run `jira-reporter` in Kubernetes

### PR DESCRIPTION
**Pull request by mixoteric**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,3 +29,32 @@ sandbox:
...
 update_classifier_config:
 	python ${project_name}/bin/update_classifier_config.py
+
+vault:
+	rm -rf docker/vault docker/secrets
+	mkdir -p docker/vault
+	docker run --interactive --tty --rm \
+		--volume "${pwd}/docker/vault:/var/lib/secrets" \
+		--env USER=${$USER} \
+		artifactory.wikia-inc.com/ops/init-vault:0.0.37 \
```
Comment (ljagiello) [inline Makefile]: I think the latest version is `0.0.40`.

File: docker/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          initContainers:
+          - args:
+            - secret/chef/jira/jira-reporter
+            env:
+            - name: VAULT_ADDR
+              value: active.vault.service.sjc.consul:8200
+            - name: ENV
+              value: prod
+            name: secrets
+            image: artifactory.wikia-inc.com/ops/init-vault:0.0.37
```
Comment (ljagiello) [inline docker/cronjob.yaml]: You can bump the version here also just to keep it the latest.

File: Makefile (hunk 1 above)
Comment (mixoteric) [reply Makefile]: Done

File: docker/cronjob.yaml (hunk 2 above)
Comment (mixoteric) [reply docker/cronjob.yaml]: Done

**Pull request merged**